requirements.txt

```diff
@@ -4,6 +4,7 @@ scipy>=0.18
 scikit-learn>=0.18
 numpy>=1.11.0
 cycler>=0.10.0
+pytest>=3.0.7
```

Comment:
We're really kind of wary of adding too many dependencies to Yellowbrick; we've had a tough time with the dependencies we do have in terms of getting a Conda package together, and dealing with different versions. At the very least, I'd ask you to move this dependency to `tests/requirements.txt` -- however, the way I see you using pytest below, there is actually already a solution using `unittest`; let me know what you think.